bin/setup

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+IFS=$'\n\t'
```

Comment:
🙇 Amended. The reason I always spell things out in code is because of the increased readability. Bash is a cryptic language by default -- and even I forget the meaning sometimes -- having the long form serves as built-in documentation without having to resort to code comments. This is also the standard I provide via  the [Bashsmith](https://alchemists.io/projects/bashsmith) project as well.